client/lib/analytics/utils.js

```diff
@@ -95,3 +101,16 @@ export function shouldSkipAds() {
 	debug( `Is Skipping Ads: ${ result }` );
 	return result;
 }
+
+/**
+ * Check if a path should report the currently selected site ID.
+ *
+ * Some paths should never report it because it's used
+ * to tell general admin and site-specific activities apart.
+ *
+ * @param {String} path The tracked path.
+ * @returns {Boolean} If the report should null `blog_id`.
+ */
+export const shouldReportOmitBlogId = path =>
```

Comment:
Should we add some quick unit tests for this function? It make help shape it a bit more easily if we provide examples.